app/views/campaigns/show.json.jbuilder

```diff
@@ -9,5 +9,31 @@ if @campaign
     json.template_description @campaign.template_description
     json.default_course_type @campaign.default_course_type
     json.default_passcode @campaign.default_passcode
+    json.courses_count @presenter.courses.count  
+    json.user_count @presenter.user_count
+    json.new_article_count_human number_to_human(@presenter.courses.sum(:new_article_count))
+    json.word_count_human number_to_human(@presenter.word_count)
+    json.references_count_human number_to_human(@presenter.references_count)
+    json.view_sum_human number_to_human(@presenter.courses.sum(:view_sum))
+    json.article_count_human number_to_human(@presenter.courses.sum(:article_count))
+    json.upload_count_human number_to_human(@presenter.courses.sum(:upload_count))
+    json.uploads_in_use_count_human number_to_human(@presenter.uploads_in_use_count)
+    json.uploads_in_use_count @presenter.uploads_in_use_count
+    json.upload_usage_count_human number_to_human(@presenter.upload_usage_count)
+    json.upload_usage_count @presenter.upload_usage_count
+    json.trained_percent_human number_to_human(@presenter.trained_percent)
+    json.course_string_prefix @presenter.course_string_prefix 
+    json.show_the_create_course_button Features.open_course_creation? && current_user&.admin?
+    json.editable @editable
+    json.register_accounts @campaign.register_accounts
+    json.current_user_admin @is_admin
+    json.requested_accounts_any @campaign.requested_accounts.any?
+    json.organizers_any @campaign.organizers.any?
+    json.organizers @campaign.organizers
+    json.current_user current_user
+    json.template_description_present @campaign.template_description.present?
```

Comment:
I think everything from current_user_admin down through template_description_present should be excluded for now. Some of this I think will ultimately be better handled on the frontend.

We will need `organizers` later on, but we shouldn't pass the entire User record for each one (as this code would do), just probably the usernames.

Reply:
I removed the lines from show.json.jbuilder file 29 to 34.